$NetBSD: patch-src_CMakeLists.txt,v 1.1 2019/07/20 23:09:27 gdt Exp $

On NetBSD, don't use -ldl (why?).

Sent upstream via email 20190427.

--- src/CMakeLists.txt.orig	2019-04-17 19:54:00.000000000 +0000
+++ src/CMakeLists.txt
@@ -146,6 +146,8 @@ endif (HAVE_GETADDRINFO_A)
 if (UNIX)
 	if (APPLE)
 		set (MOSQ_LIBS ${MOSQ_LIBS} dl m)
+	elseif (${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
+		set (MOSQ_LIBS ${MOSQ_LIBS} m)
     elseif(QNX)
         set(MOSQ_LIBS ${MOSQ_LIBS} m socket)
     else(APPLE)
